_includes/links.md

```diff
@@ -16,6 +16,7 @@
 [email]: mailto:team@carpentries.org
 [filters]: https://jekyllrb.com/docs/liquid/filters/
 [ghp-dependencies]: https://pages.github.com/versions/
+[git]: https://git-scm.com/
```

Comment:
I guess we should link to the git homepage, but please also find a way to add a link to https://swcarpentry.github.io/git-novice/ in this section

Reply:
Added.